Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -37,8 +37,6 @@ public class RepositoryPackageContainerProvider: PackageContainerProvider {
 
     public func getContainer(for identifier: RepositorySpecifier) throws -> Container {
```

Comment:
I removed the protocol requirement but Workspace uses this method https://github.com/apple/swift-package-manager/pull/830/files#diff-16a17153d6f335742f5d4039edf8be5eR569
We can either keep this or make that call sync using condition, I'd rather keep this because the protocol requirement is gone anyway.
What do you think?